README.md

```diff
@@ -100,7 +100,9 @@ public class Example {
 
 ### TLS
 
-The HTTP server implements TLS 1.0-1.3 using the Java SSLEngine. To enable TLS for your server, you need to create an `HTTPListenerConfiguration` that includes a certificate and private key. Most production use-cases will use a proxy such as Apache, Nginx, ALBs, etc. In development, it is recommended that you set up self-signed certificates and load those into the HTTP server.
+The HTTP server implements TLS 1.0-1.3 using the Java SSLEngine. To enable TLS for your server, you need to create an
```

Comment:
Generally speaking we don't add line breaks and just use soft-wrap in an editor if we want a multi-line view.